### PR TITLE
Minor wording (ko)

### PR DESCRIPTION
**Pull request by marocchino**
@yous review please.

### CODE REVIEW

File: i18n/ko.json [hunk 1]
```diff
@@ -26,12 +26,12 @@
...
-      "missing_file": "에러: {{{exerciseFile}}}은(는) 존재하지 않습니다!",
-      "not_a_workshopper": "에러: {{{exerciseFile}}}은(는) 워크숍의 연습문제가 아닙니다.",
+      "missing_file": "오류: {{{exerciseFile}}}은(는) 존재하지 않습니다!",
+      "not_a_workshopper": "오류: {{{exerciseFile}}}은(는) 워크숍의 연습문제가 아닙니다.",
       "preparing": "연습문제의 준비에 실패했습니다. {{{err}}}",
-      "loading": "연습문제의 택스트를 불러오는 중 에러가 발생했습니다. {{{err}}}"
+      "loading": "연습문제의 택스트를 불러오는 중 오류가 발생했습니다. {{{err}}}"
     },
-    "no_uncomplete_left": "지금의 연습문제를 마지막으로 더 이상 연습문제가 없습니다.",
+    "no_uncomplete_left": "지금 연습문제를 마지막으로 더 이상 연습문제가 없습니다.",
```
Comment (yous) [inline i18n/ko.json]: How about '이' instead of '지금'?

**Comment by yous** [comment]
I prefer '연습 문제' rather than '연습문제'. How about you?

File: i18n/ko.json [hunk 2]
```diff
@@ -26,12 +26,12 @@
       "none_active": "풀고 있는 연습문제가 없습니다. 메뉴에서 골라주세요.",
       "missing": "{{{name}}}(이)라는 연습문제는 없습니다.",
       "unexpected_error": "{{mode}}을(를) 할 수 없었습니다. {{{err}}}",
-      "missing_file": "에러: {{{exerciseFile}}}은(는) 존재하지 않습니다!",
-      "not_a_workshopper": "에러: {{{exerciseFile}}}은(는) 워크숍의 연습문제가 아닙니다.",
+      "missing_file": "오류: {{{exerciseFile}}}은(는) 존재하지 않습니다!",
+      "not_a_workshopper": "오류: {{{exerciseFile}}}은(는) 워크숍의 연습문제가 아닙니다.",
       "preparing": "연습문제의 준비에 실패했습니다. {{{err}}}",
-      "loading": "연습문제의 택스트를 불러오는 중 에러가 발생했습니다. {{{err}}}"
+      "loading": "연습문제의 택스트를 불러오는 중 오류가 발생했습니다. {{{err}}}"
```
Comment (yous) [inline i18n/ko.json]: 택스트 → 텍스트

**Comment by marocchino** [comment]
Well, It's will be huge across repos. but, OK.

File: i18n/ko.json (hunk 2 above)
Comment (yous) [inline i18n/ko.json]: How about '연습문제의 글을 불러오는 중' or '연습문제를 불러오는 중'?

**Comment by marocchino** [comment]
@yous It is over?

**Comment by yous** [comment]
I want to remove spaces between '{{count}}' or '{{amount}}' and '개'.

**Comment by marocchino** [comment]
Applied.

**Comment by yous** [comment]
I'm done. :relaxed:

**Comment by marocchino** [comment]
Thank you!

**Pull request merged**